client/src/app/ui/modules/search-selector/components/repo-search-selector/repo-search-selector.component.ts

```diff
@@ -53,6 +53,11 @@ export class RepoSearchSelectorComponent extends BaseSearchSelectorComponent imp
     ) {
         super(ngControl);
         this.shouldPropagateOnRegistering = false;
+        this.noneItem = {
+            getListTitle: (): string => this.noneTitle,
+            getTitle: (): string => this.noneTitle,
+            id: null
+        };
```

Comment:
Why do you need this here? Does this affect any other usages of this component?

Reply:
I needed it here, to set noneItem.id to 'null' instead of '0'. I found no other usage of none item with the repo-search-selector.

Reply:
I am asking because of https://github.com/OpenSlides/openslides-client/pull/3272

This is indeed used for example in `participant-create-wizard.component.html` and `participant-detail-view.component.html`.

Reply:
Okay, how can I use the repo-search-selector with noneItem, that it doesn't deliver a 0 to the action?

Reply:
I have removed the noneItem change and set 0 to null before sending the payload to the repos.